src/results/financial.jl

```diff
@@ -183,8 +200,10 @@ function initial_capex(m::JuMP.AbstractModel, p::REoptInputs; _n="")
 
             if option[2].heat_pump_configuration == "WSHP"
                 initial_capex += option[2].installed_cost_per_kw[2]*option[2].heatpump_capacity_ton*value(m[Symbol("binGHP"*_n)][option[1]])
+                initial_capex -= value(m[:AvoidedCapexByGHP])
```

Comment:
This could be included just once outside of the if statements because it defaults to 0, right?

Reply:
Yes, moved up to the beginning, same with m[:AvoidedCapexByASHP]